### PR TITLE
Added Compare Two Strings Represented As Linked Lists challenge and solution

### PR DESCRIPTION
**Pull request by ayunav**

### CODE REVIEW

File: challenges/LinkedLists/CompareTwoStrings/CompareTwoStringsRepresentedAsLinkedLists.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+##Compare two strings represented as linked lists
+
+The following problem is taken from [geeksforgeeks.org](http://www.geeksforgeeks.org/compare-two-strings-represented-as-linked-lists/). NB: the link contains solution. We recommend you to try this yourself first.
+
+Given two linked lists, represented as linked lists (every character is a node in linked lists), write a function compare() that works similar to strcmp(), i.e., it returns 0 if both strings are same, 1 if first linked list is lexicographically greater, and -1 if second string is lexicographically greater.
```
Comment (sarahzinger) [inline challenges/LinkedLists/CompareTwoStrings/CompareTwoStringsRepresentedAsLinkedLists.md:5]: @ayunav this is great! would you be interested in maybe giving a two sentence description of what "lexicographically greater" means or giving a link? Thanks!

**Comment by sarahzinger** [comment]
@ayunav thank you so much for submitting a problem! it's really awesome 💯 !!! If you have the time I made one tiny comment for something that I think could use a little bit more explanation. But if not, we can totally merge it as is! Thanks again!

File: challenges/LinkedLists/CompareTwoStrings/CompareTwoStringsRepresentedAsLinkedLists.md (hunk 1 above)
Comment (ayunav) [inline challenges/LinkedLists/CompareTwoStrings/CompareTwoStringsRepresentedAsLinkedLists.md:5]: Hi Sarah, I added a definition for lexicographical order. Hope this helps!

**Comment by sarahzinger** [comment]
Looks awesome @ayunav  thanks so much!

**Comment by ayunav** [comment]
👍  😊   Thanks for reviewing and merging!

**Pull request merged**